OpenEdX/Managers/PushNotificationsManager/PushNotificationsManager.swift

```diff
@@ -9,56 +9,58 @@ import Foundation
 import Core
 import UIKit
 import Swinject
```

Comment:
I guess we are safe to remove this import

Reply:
👍